test/libdnf5/repo/test_temp_files_memory.cpp

```diff
@@ -79,6 +79,15 @@ void TempFilesMemoryTest::test_add_files_when_empty_storage() {
     CPPUNIT_ASSERT_EQUAL(new_paths, memory.get_files());
 }
 
+void TempFilesMemoryTest::test_add_no_files_when_empty_storage() {
```

Comment:
May I ask you to add additional information to test - what is in commit message is perfect - test that it does not crash? Without that information the test looks like that it is not required.

Reply:
I've added a comment there. Is it OK like that?

Reply:
And thanks for the suggestion!

Reply:
Thank you